src/calculateRentalCost.js

```diff
@@ -22,13 +23,29 @@
  *  - calculateRentalCost(3) === 100
  *  - calculateRentalCost(7) === 230
  *
- * @param {number} days
+ * @param {number} payTotal
  *
  * @return {number}
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const payOneDay = 40;
+  const discountMore3Days = 20;
+  const discountMore7Days = 50;
+
+  const payTotal = days * payOneDay;
+  const rentMore3Days = payTotal - discountMore3Days;
+  const rentMore7Days = payTotal - discountMore7Days;
```

Comment:
Don't use hardcoded numbers in your variable names. If, for example, it is decided in future to give small discont after 4+ days instead, you will need not only to change condition, but also fix this variable's name everywhere